From 150cccb4a935a2a889ac8379e08dcaa52e48f6b5 Mon Sep 17 00:00:00 2001
From: Kai Koehne <kai.koehne@qt.io>
Date: Tue, 22 Jan 2019 16:16:29 +0100
Subject: [PATCH 1/6] windeployqt: Fix clang warning about switch statement

warning: unannotated fall-through between switch labels
insert 'break;' to avoid fall-through

Change-Id: I1aeae4233898a59eac9872485ec12cde2278ade5
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
---
 src/windeployqt/main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/windeployqt/main.cpp b/src/windeployqt/main.cpp
index 2bb1ad95..da61f9b8 100644
--- a/src/windeployqt/main.cpp
+++ b/src/windeployqt/main.cpp
@@ -1140,6 +1140,7 @@ static QStringList compilerRunTimeLibs(Platform platform, bool isDebug, unsigned
         }
         result.append(redistFiles);
     }
+        break;
     default:
         break;
     }
-- 
2.17.1

